src/app/components/common/node-editor/custom-connection/custom-connection.component.html

```diff
@@ -2,10 +2,8 @@
   <path [attr.d]="path" class="hover-path"></path>
   <path [attr.d]="path" #svgpath [class]="{ selected: data.selected }" />
 </svg>
-<!-- Show label on the connection line -->
-<!-- <span (pointerdown)="$event.stopPropagation()" class="connection-status label has-text-success" [style]="{
-    left: point.x + 'px',
-    top: point.y + 'px',
-  }">
-  <i class="bi bi-vinyl"></i>
+<!-- Show ban icon on the connection wire, if egress/ingress disabled -->
```

Comment:
Remove this and instead show a most recent reading icon with popover having reading at each stage?